app/views/corporate_bodies/show.html.haml

```diff
@@ -1,36 +1,11 @@
-%p#notice= notice
+.container
+  %p#notice= notice
 
-%p
-  %b Name:
-  = @corporate_body.name
-%p
-  %b Alternate names:
-  = @corporate_body.alternate_names
-%p
-  %b Location:
-  = @corporate_body.location
-%p
-  %b Inception:
-  = @corporate_body.inception
-%p
-  %b Inception year:
-  = @corporate_body.inception_year
-%p
-  %b Dissolution:
-  = @corporate_body.dissolution
-%p
-  %b Dissolution year:
-  = @corporate_body.dissolution_year
-%p
-  %b Wikidata uri:
-  = @corporate_body.wikidata_uri
-%p
-  %b Viaf:
-  = @corporate_body.viaf_id
-%p
-  %b Comments:
-  = @corporate_body.comments
+.btn-group
+  = link_to 'Edit', edit_corporate_body_path(@corporate_body), class: 'btn btn-default'
+  = link_to 'Back', corporate_bodies_path, class: 'btn btn-outline-secondary'
 
-= link_to 'Edit', edit_corporate_body_path(@corporate_body)
-\|
-= link_to 'Back', corporate_bodies_path
+- CorporateBodiesController::ATTRIBUTES_TO_SHOW.each do |attr|
+  %p
+    %b #{CorporateBody.human_attribute_name(attr)}:
```

Comment:
This method takes attribute name from i18n files specific for given model (active_record.he.yml)